operations/template/alert.tf

```diff
@@ -0,0 +1,42 @@
+resource "azurerm_monitor_action_group" "monitor" {
+  name                = "cdcti${var.environment}-actiongroup"
+  resource_group_name = data.azurerm_resource_group.group.name
+  short_name          = "cdcti-alerts"
+
+  email_receiver {
+    name          = "cdcti-flexion-slack-email-receiver"
+    email_address = var.alert_slack_email
+  }
+}
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "alert" {
+  name                = "cdcti-${var.environment}-api-log-token-alert"
+  location            = data.azurerm_resource_group.group.location
+  resource_group_name = data.azurerm_resource_group.group.name
+
+  action {
+    action_group  = [azurerm_monitor_action_group.monitor.id]
+    email_subject = "FATAL: The access token has expired!"
+  }
+
+  data_source_id = azurerm_linux_web_app.api.id
+  description    = "Alert when total results cross threshold"
+  enabled        = true
+
+  query = <<-QUERY
+      AppServiceConsoleLogs
+      | where ResultDescription has "FATAL: The access token has expired."
+      and TimeGenerated >= ago(30m)
+      and TimeGenerated <= now()
+      | summarize count()
+    QUERY
+
+  severity    = 3
+  frequency   = 10
+  time_window = 30
+
+  trigger {
+    operator  = "GreaterThan"
+    threshold = 5
+  }
```

Comment:
This is saying that the log needs to show up 5 times within the last 30 minutes?  Or...?

Reply:
Correct, open to thoughts on this check. Right now, the log is stateless so it will fire an alert  every time this check counts 5 of these logs in the last 30 minutes.

Reply:
Now updated to fail on 1 log search found.